bottlerocket-settings-models/settings-extensions/host-containers/src/lib.rs

```diff
@@ -38,6 +38,7 @@ struct HostContainer {
     enabled: bool,
     superpowered: bool,
     user_data: ValidBase64,
+    command: Vec<String>,
```

Comment:
These might be safer as `Vec<SingleLineString>` - the `toml_encode` helper you're using over in the templates where the value is rendered makes this safe, but in theory the field could be used by some other template that might not expect newlines.